### ISSUE TITLE
Unable to generate schemas due to swift-format version mismatch.

### ISSUE THREAD

**Issue by thingdeux** [maticzav/swift-graphql#129]
I am unable to generate schemas using Swift-GraphQL v5.0.0 due to swift-format version mismatch.

(Using Swift-GraphQL v5 - installed via Homebrew)

1.  Run the swift-graphql codegen command.
ex: swift-graphql <gateway> --config <location> -- output <location> --header <OAuth header>

### Expected behavior
Swift-GraphQL based schema is generated.

### Actual behavior
I receive the error below re: Swift-Format version mismatch

- OS: MacOS Ventura 13.3.1 (a) - M1 CPU

### Log Output
```
✔ Schema loaded!
✖ The operation couldn’t be completed. (SwiftFormat.SwiftFormatError error 2.)
Error: There was an error formatting the code. Make sure your Swift version (i.e. `swift-format`) matches the `swift-format` version. If you need any help, don't hesitate to open an issue and include the log above!
```

**Comment by thingdeux** [comment]
Hmmm, looks like there was some breaking change introduced with Swift 5.8 🤔

(From Swift-Formats Readme)
![image](https://user-images.githubusercontent.com/910846/236265467-a0e2b2fa-89e7-45ae-b7b2-02d35e64db64.png)

**Comment by maticzav** [comment]
What's your local Swift version?

**Comment by thingdeux** [comment]
> What's your local Swift version?

@maticzav 

```bash
❯ swift --version
swift-driver version: 1.75.2 Apple Swift version 5.8 (swiftlang-5.8.0.124.2 clang-1403.0.22.11.100)
Target: arm64-apple-macosx13.0
```

**Comment by thingdeux** [comment]
Alrighty - so, I found the issue.  Pulled down the code and built it myself to see what was going on.

@maticzav 
In short, looks like a deprecation message was generated that didn't terminate itself.


ex:

```swift
@available(*, deprecated, message: "Fetch this using "note" instead")
```

The "note" inside of a string built with "" threw an exception with swift-format
I terminated these w/

```swift
@available(*, deprecated, message: "Fetch this using \"note\" instead")
```

And swift-format functioned without issue.
